parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -298,8 +299,14 @@ public EncodingStats convertEncodingStats(List<PageEncodingStats> stats) {
     return formatStats;
   }
 
+  @Deprecated
```

Comment:
Since this is a public API, please add a deprecation warning in Javadoc with a link to the alternative.

Reply:
Don't forget to include "will be removed in 2.0.0" because those help us know when we can actually remove the deprecated methods.